common/main/af.xml

```diff
@@ -6160,7 +6160,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			</compoundUnit>
 			<unit type="acceleration-g-force">
 				<displayName>swaartekrag</displayName>
-				<unitPattern count="one">{0} swaartekrag</unitPattern>
+				<unitPattern count="one">↑↑↑</unitPattern>
```

Comment:
this is the same as in the previous PR, where we observed it's wrong since it's replacing a hard value with an inheritance marker and in a path that didn't allow changes

how to translate this insight into fixing the code, remains unclear to me...

Reply:
The path is` //ldml/units/unitLength[@type="long"]/unit[@type="acceleration-g-force"]/unitPattern[@count="one"]`

↑↑↑ resulted from two votes in v42 for "{0} swaartekrag"; while these were auto-imported in v43, the inconsistent behavior of getBaileyValue resulted in "{0} swaartekrag" being seen as equal to the bailey value and therefore replaced by ↑↑↑, although previously and subsequently the same code returned "{0} G" as bailey for the same locale and path

Reply:
As seen by "modify -fR", the baseline value is "{0} swaartekrag" and the plain vxml value is "↑↑↑"

Reply:
I revised the code to get baselineValue from UNRESOLVED CLDRFile -- but this doesn't change

Reply:
As it turns out,  "modify -fR" finds the baseline value is "↑↑↑", NOT "{0} swaartekrag" -- at least if the unresolved file is used -- af.xml has "↑↑↑"

Therefore wantRevertToBaseline returns false per criterion 1